app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -35,9 +35,12 @@ def custom_index_relation(collection)
     def_param_group :repo do
       param :url, String, :desc => N_("repository source url")
       param :gpg_key_id, :number, :desc => N_("id of the gpg key that will be assigned to the new repository")
-      param :ssl_ca_cert_id, :number, :desc => N_("Idenifier of the SSL CA Cert")
-      param :ssl_client_cert_id, :number, :desc => N_("Identifier of the SSL Client Cert")
-      param :ssl_client_key_id, :number, :desc => N_("Identifier of the SSL Client Key")
+      param :ssl_ca_cert_id, :number, :desc => N_("Identifier of the SSL CA Cert content credential")
+      param :ssl_client_cert_id, :number, :desc => N_("Identifier of the SSL Client Cert content credential")
+      param :ssl_client_key_id, :number, :desc => N_("Identifier of the SSL Client Key content credential")
+      param :ssl_ca_cert_name, String, :desc => N_("Name of the SSL CA Cert content credential")
+      param :ssl_client_cert_name, String, :desc => N_("Name of the SSL Client Cert content credential")
+      param :ssl_client_key_name, String, :desc => N_("Name of the SSL Client Key content credential")
```

Comment:
hrmmm, we normally solve this in hammer via a name resolver.  Any reason to not do that here?

Reply:
Thats probably a better way to go about this, I totally forgot about that aspect of hammer. :see_no_evil: I'll open up a PR there!